docs/zh_cn/tutorials/customize_dataset.md

```diff
@@ -75,31 +75,47 @@ _base_ = './cascade_mask_rcnn_r50_fpn_1x_coco.py'
 # 1. 数据集设定
 dataset_type = 'CocoDataset'
 classes = ('a', 'b', 'c', 'd', 'e')
-data = dict(
-    samples_per_gpu=2,
-    workers_per_gpu=2,
-    train=dict(
+data_root='path/to/your/'
+
+train_dataloader = dict(
+    batch_size=2,
+    num_workers=2,
+    dataset=dict(
         type=dataset_type,
-        # 将类别名字添加至 `classes` 字段中
-        classes=classes,
-        ann_file='path/to/your/train/annotation_data',
-        img_prefix='path/to/your/train/image_data'),
-    val=dict(
+        # 将类别名字添加至 `metainfo` 字段中
+        metainfo=dict(CLASSES=classes),
+        data_root=data_root,
+        ann_file='train/annotation_data',
+        data_prefix=dict(img='train/image_data')
+    )
+
+val_dataloader = dict(
+    batch_size=2,
+    num_workers=2,
+    dataset=dict(
         type=dataset_type,
-        # 将类别名字添加至 `classes` 字段中
-        classes=classes,
-        ann_file='path/to/your/val/annotation_data',
-        img_prefix='path/to/your/val/image_data'),
-    test=dict(
+        # 将类别名字添加至 `metainfo` 字段中
+        metainfo=dict(CLASSES=classes),
```

Comment:
I think we maybe need to add  `test_mode=True` in val and test dataloader, same as english docs.